test/without-vue.js

```diff
@@ -0,0 +1,14 @@
+import VueSelector from '../lib';
```

Comment:
Don't need a separate page.
Just use a build-in `about:blank`. Also don't need to `Selector`.
```js
fixture `page without vue`;

test('there is no Vue on the tested page', async t => {
    const listVueSelector = VueSelector('list');

    await t.expect(listVueSelector.count).eql(0)
});
```